packages/headless/scripts/extract-documentation.mjs

```diff
@@ -10,9 +10,13 @@ async function main() {
     .map((file) => `${dir}/${file}`);
   console.log('extracting documentation using', paths);
 
-  const jobs = paths.map((path) =>
-    execute('api-extractor', ['run', '-l', '-c', path])
-  );
+  const jobs = paths.map((path) => {
+    const apiProcess = execa('api-extractor', ['run', '-l', '-c', path], {});
```

Comment:
`execa` seems to work a lot like `process.spawn`, where it returns you the streams instead of a combined/final output.

I added `exec.mjs` to make it easy to execute a command, pipe the output to the console for debugging, and return the combined output (or throw). If this isn't too time-sensitive, I think we should try to replace `exec.mjs`'s `process.spawn` with execa.

Reply:
Agree that'd be nice eventually. For now, because there's a lot of change happening, I wanted to have the smallest footprint possible.